### ISSUE TITLE
Support resource_manager_tags on instance_template module

### ISSUE THREAD

**Issue by kpettijohn** [terraform-google-modules/terraform-google-vm#395]
### TL;DR

I might have missed it but seems like the `instance_template` module doesn't support setting [`resource_manager_tags`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_instance_template#resource_manager_tags). 

### Terraform Resources

Resource: [`google_compute_instance_template`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_instance_template#resource_manager_tags)
Module: [`instance_template`](https://github.com/terraform-google-modules/terraform-google-vm/tree/master/modules/instance_template)

### Detailed design

```markdown
variable "resource_manager_tags" {
  type        = map(string)
  default     = {}
}

module "vm_instance_template" {
  source  = "terraform-google-modules/vm/google//modules/instance_template"
  ...
  resource_manager_tags = var.resource_manager_tags
}
```

### Additional information

I would like to use resource manager tags to support using IAM conditionals.
